doc/getting-started/apl-nuc.rst

```diff
@@ -47,41 +47,50 @@ complete this setup.
    and follow the `Clear Linux installation guide
    <https://clearlinux.org/documentation/clear-linux/get-started/bare-metal-install>`__
    as a starting point for installing Clear Linux onto your platform.  Follow the recommended
-   options for choosing an **Automatic** installation type, and using the platform's
+   options for choosing an **Manual** installation type, and using the platform's
    storage as the target device for installation (overwriting the existing data
    and creating three partitions on the platform's storage drive).
+   
+   High-level steps should be: 
+   
+   #.  Install Clear on a NUC Use the "Manual" option.
+   #.  Use default partition scheme for storage
+   #.  Name the host "clr-sos-guest"
+   #.  Add an administrative user "clear" with "sudoers" privilege
```

Comment:
@gvancuts Are you OK with telling folks **what** to do but not **how** to do it?  (Do we need the actual commands here to show how to "Add an administrative user with sudoers privilege", and some of these other steps?)

Reply:
I haven't tried it but if how the steps to do so via the `Manual (Advanced)` installation option are obvious, we can leave it as it is. Maybe, make it clearer that the manual option is from the Clear Linux installer (and is called `Manual (Advanced)`).